airbyte-server/src/main/java/io/airbyte/server/apis/ConfigurationApi.java

```diff
@@ -633,16 +627,6 @@ public WebBackendConnectionRead webBackendUpdateConnection(final WebBackendConne
     return execute(() -> webBackendConnectionsHandler.webBackendUpdateConnection(webBackendConnectionUpdate));
   }
 
-  @Override
-  public SourceRead webBackendCreateSource(final SourceCreate sourceCreate) {
```

Comment:
why remove these endpoints?

Reply:
it's merged into the completeOAuth API call to all the outputs related to oauth are "injected" together back to the API.

The "runtime" injection logic would only need to be done in the scheduler (not in the API anymore), so we don't really need the webbackend